sdk/core/amqp-common/rollup.test.config.js

```diff
@@ -3,5 +3,14 @@
 
 import * as base from "./rollup.base.config";
 
-// Node tests are run via ts-node
-export default [base.browserConfig(true)];
+const inputs = [];
+
+if (!process.env.ONLY_BROWSER) {
+  inputs.push(base.nodeConfig({ test: true }));
+}
+
+if (!process.env.ONLY_NODE) {
+  inputs.push(base.browserConfig({ test: true }));
```

Comment:
`base.browserConfig`  takes a single boolean argument. Why are we passing a json object?